### PR TITLE
Resprites Advanced Burn and Trauma Kits

### PR DESCRIPTION
**Pull request by CameronWoof**
New sprites for the advanced burn and trauma kits, including support for indicating how many are left in the stack. Made by yours truly.

https://cdn.discordapp.com/attachments/419883780275175426/604512226811838465/unknown.png

### CODE REVIEW

**Comment by Wickedtemp** [comment]
I like it!

File: code/game/objects/items/stacks/medical_vr.dm [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+	icon = 'icons/obj/stacks_vr.dmi'
+
+/obj/item/stack/medical/advanced/New()
+	..()
+	update_icon()
+	return
+
+/obj/item/stack/medical/advanced/update_icon()
+	if(amount <= 2)
+		icon_state = initial(icon_state)
```
Comment (chaoko99) [inline code/game/objects/items/stacks/medical_vr.dm]: Make all this into a switch statement. Use `[lower bound] to [upper bound]`. It's just faster.

File: code/game/objects/items/stacks/medical_vr.dm [hunk 2]
```diff
@@ -0,0 +1,21 @@
+/obj/item/stack/medical/advanced
+	icon = 'icons/obj/stacks_vr.dmi'
+
+/obj/item/stack/medical/advanced/New()
+	..()
```
Comment (chaoko99) [inline code/game/objects/items/stacks/medical_vr.dm]: change this into an `Initialize()`

Also why the return?

File: code/game/objects/items/stacks/medical_vr.dm [hunk 3]
```diff
@@ -1,21 +1,21 @@
 /obj/item/stack/medical/advanced
 	icon = 'icons/obj/stacks_vr.dmi'
 
-/obj/item/stack/medical/advanced/New()
+/obj/item/stack/medical/advanced/Initialize()
 	..()
```
Comment (chaoko99) [inline code/game/objects/items/stacks/medical_vr.dm]: `. = ..()` 
not ..()

**Comment by KasparoVy** [comment]
Looks sweet, intuitive design.

**Comment by SpadesNeil** [comment]
Please send a version of this to Polaris.

**Pull request merged**